src/components/indicator/clearHistoryDialog.ts

```diff
@@ -1,7 +1,7 @@
 import Clutter from '@girs/clutter-16';
 import type { ExtensionBase } from '@girs/gnome-shell/dist/extensions/sharedInternals';
 import { MessageDialogContent } from '@girs/gnome-shell/dist/ui/dialog';
-import { ModalDialog } from '@girs/gnome-shell/dist/ui/modalDialog';
+import ModalDialog from '@girs/gnome-shell/dist/ui/modalDialog';
```

Comment:
```suggestion
import { ModalDialog } from '@girs/gnome-shell/dist/ui/modalDialog';
```

That module exports a number of different items, but no default export.